ansible/library/reduce_and_add_sonic_images.py

```diff
@@ -82,7 +84,6 @@ def install_new_sonic_image(module, new_image_url):
         exec_command(module,
                      cmd="mount -t tmpfs -o size=1300M tmpfs /tmp/tmpfs",
```

Comment:
As `save_as` is now input to this function, I think we probably should assume it can be any valid path (although we do not have such a scenario now). If we assume so, the creation of the tmpfs partition looks problematic: we will mount `/tmp/tmpfs` as long as the input is not `/host/downloaded-sonic-image`, even if the input is not related to `/tmp/tmpfs`. I would suggest doing this mount only when `save_as` starts with `/tmp/tmpfs`.

Reply:
Got it. Improved the check accordingly. Thanks!